static/js/actions.js

```diff
@@ -28,5 +29,6 @@ export default function hydrate() {
     dispatch(observingRunsActions.fetchObservingRuns());
     dispatch(telescopesActions.fetchTelescopes());
     dispatch(taxonomyActions.fetchTaxonomies());
+    dispatch(groupActions.fetchGroup(7));
```

Comment:
Can you explain why we need this here? And this wouldn't work in production either way since groupID 7 doesn't have a special meaning right?

Reply:
Ah woops, this was a left-behind debugging-only addition. Removed :)

Reply:
I spent a few hours tracking down a typo in one of the reducers and added this to help pin down the issue